flexmeasures/data/models/charts/belief_charts.py

```diff
@@ -50,9 +51,23 @@ def bar_chart(
     if sensor.event_resolution == timedelta(0) and sensor.has_attribute("interpolate"):
         mark_type = "area"
         mark_interpolate = sensor.get_attribute("interpolate")
+    if chart_type == "histogram":
+        description = "A histogram showing the distribution of sensor data."
+        x = {
+            **event_value_field_definition,
+            "bin": {"step": 2},
```

Comment:
Binning nicely, in a general way, for a myriad of data settings is quite tricky. I tested some cases (a price sensor, a V2G power sensor and a SoC sensor) and found the vega-lite defaults to yield more pleasant results.

```suggestion
            "bin": True,
```